pkg/util/sys/device.go

```diff
@@ -307,10 +326,11 @@ func CheckIfDeviceAvailable(executor exec.Executor, name string) (int, bool, str
 	}
 
 	// check if there is a file system on the device
-	devFS, err := GetDeviceFilesystems(name, executor)
+	/* devFS, err := GetDeviceFilesystems(name, executor)
```

Comment:
i'd expect this is still needed for old-style OSDs.

Reply:
This is addressed now.  GetDeviceFileSystems is called when PVCs are on Nodes. We are using `pvcBacked` flag which if true won't call GetDeviceFileSystems method because `udev` is failing on PVC block device. It sill set `devFs` to empty string for PVC backed OSD.